editor/resources/coronest.lua

```diff
@@ -0,0 +1,80 @@
+-- The MIT License (MIT)
```

Comment:
Copied from https://github.com/saucisson/lua-coronest, the only change is addition of a License (MIT) in a comment.